thirdparty/entry.inc.php

```diff
@@ -48,3 +49,4 @@
 add_action( 'litespeed_init', 'LiteSpeed\Thirdparty\NextGenGallery::preload' );
 add_action( 'litespeed_init', 'LiteSpeed\Thirdparty\AMP::preload' );
 add_action( 'litespeed_init', 'LiteSpeed\Thirdparty\Elementor::preload' );
+add_action( 'litespeed_init', 'LiteSpeed\Thirdparty\Gravity_Forms::preload' );
```

Comment:
You can't define both `detect` and `preload` but only have `preload` defined in the following class.